src/protos/stub.proto

```diff
@@ -0,0 +1,174 @@
+package bicyclade;
+
+message Container {
+  optional PlayerAction player_action;
+  optional ServerAction server_action;
+  optional SpectatorAction spectator_action;
+}
+
+message Game {
+  optional uint32 id;
+  repeated Player players;
+  repeated Card cards;
```

Comment:
I'm not clear on this one: I was thinking of having a list of cards for the discarded pile, list of cards to pick, and a list of cards on the board ?
what is your list representing ?

Reply:
Yes I forgot the discarded cards. This list is supposed to represent the cards on the board. I don't think think the cards to pick should be communicated to the clients.